common/source/docs/common-simplebgc-gimbal.rst

```diff
@@ -22,28 +22,28 @@ Although the SimpleBGC can be connected using PWM (similar to the Tarot gimbal)
 
 In ArduPilot/APM Planner/Mission planner set the following variables:
 
-- :ref:`MNT_TYPE <MNT_TYPE>` to 4 / "Mount Type (None, Servo or MavLink)"
-- :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` to 1 / "MavLink" (Notee "SERIAL2" should be "SERIAL1" if using Telem1 port, SERIAL4 if using Serial4/5, etc)
+- :ref:`MNT1_TYPE <MNT1_TYPE>` to 4 / "Mount Type (None, Servo or MavLink)"
+- :ref:`SERIAL2_PROTOCOL <SERIAL2_PROTOCOL>` to 1 / "MavLink" (Note "SERIAL2" should be "SERIAL1" if using Telem1 port, SERIAL4 if using Serial4/5, etc)
 - :ref:`SR2_EXTRA1 <SR2_EXTRA1>` to 20
 - :ref:`SR2_POSITION <SR2_POSITION>` to 10
 - :ref:`SR2_RC_CHAN <SR2_RC_CHAN>` to 20 and all other SR2_* variables to 0.
 
 If you wish to control the pitch angle manually you can set:
 
-- :ref:`MNT_RC_IN_TILT <MNT_RC_IN_TILT>` to 6
+- :ref:`RCx_OPTION <RC1_OPTION>` to 213.
```

Comment:
```suggestion
-  ``RCx_OPTION`` to 213.
```
I dont like using a link to a specific param when discussing a general group....either just unlink and highlight, or put a "if using RC 6..etc."

Reply:
That's the best way I can think of to link to one instance of the parameter list. IMO it is better than not linking to the additional documentation about the parameter.

Reply:
I disagree...it has caused confusion in the past and I have had to answer questions about "do I really need to use that RC channel?"